.github/workflows/bower_publish.yml

```diff
@@ -37,5 +37,4 @@ jobs:
           git config user.email github-actions@github.com
           git add .
           git commit -m $TAG
-          git tag -a $TAG -m $TAG
-          git push origin master --tags
+          git push origin master
```

Comment:
No more tags?

Reply:
I'm trying to minimize the variation across bower and gh pages because they should be very nearly identical, so this is part of that

Reply:
but we should keep them :+1: